rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -0,0 +1,896 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hsearch" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis for migration from Hibernate Search 4.x to Hibernate Search 5.x.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.4.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
+        </dependencies>
+        <sourceTechnology id="hibernate-search" versionRange="[4,)" />
+        <targetTechnology id="hibernate-search5" versionRange="[5,)" />
+        <targetTechnology id="eap7" versionRange="[7,)" />
+        <tag>reviewed-2016-04-12</tag>
+    </metadata>
+    <rules>
+        <rule id="hsearch-00000">
+            <when>
+                <javaclass references="org.hibernate.search.{*}" />
+            </when>
+            <perform>
+                <classification title="Hibernate Search usage" effort="0" severity="mandatory">
+                    <description>Hibernate Search API used.</description>
+                    <tag>hibernate-search</tag>
+                </classification>
+            </perform>
+        </rule>
+        <rule id="hsearch-00001">
+            <when>
+                <javaclass references="org.hibernate.search.engine.impl.SearchMappingBuilder" />
+            </when>
+            <perform>
+                <hint title="Removed org.hibernate.search.engine.impl.SearchMappingBuilder class in Hibernate Search 5" effort="1" severity="mandatory">
```

Comment:
Similarly to other rules, this should be "Hibernate Search 5 package change for SearchMappingBuilder".